Dockerfile

```diff
@@ -1,7 +1,34 @@
 FROM python:3.6
 
 # system maintenance
-RUN apt-get update && apt-get install -y gcc r-base
+RUN apt-get update
+
+# install dependencies needed for setting up R
+RUN apt-get install -y lsb-release dirmngr gnupg apt-transport-https ca-certificates software-properties-common
+
+# get the Linux distro info, use this to set the right R download
+RUN lsb_release -a
+
+# set up the key for adding the R repo
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 95C0FAF38DB3CCAD0C080A7BDC78B2DDEABC47B7
+
+# add the correct Linux R repo
+RUN add-apt-repository 'deb https://cloud.r-project.org/bin/linux/debian bullseye-cran40/'
+
+# ensure the previous command added the right line to the sources.list files
+RUN cat /etc/apt/sources.list
```

Comment:
Some of the stuff in this dockerfile looks like it's mostly for testing, and not needed now that it's working, correct?

Reply:
I thought it would be helpful to keep these around for debugging purposes, but since we dump everything to `/dev/null` anyways (to prevent output log errors), I think we can remove them.

Reply:
Okay cool, whatever you prefer